gulpfile.js

```diff
@@ -1157,6 +1169,66 @@ gulp.task('importl10n', function(done) {
   locales.downloadL10n(L10N_DIR, done);
 });
 
+gulp.task('gh-pages-prepare', ['web-pre'], function () {
+  console.log();
+  console.log('### Creating web site');
+
+  rimraf.sync(GH_PAGES_DIR);
+
+  // 'vinyl' because web/viewer.html needs its BOM.
+  return merge([
+    vinyl.src(GENERIC_DIR + '**/*', {base: GENERIC_DIR, stripBOM: false})
+         .pipe(gulp.dest(GH_PAGES_DIR)),
+    gulp.src([FIREFOX_BUILD_DIR + '*.xpi',
+              FIREFOX_BUILD_DIR + '*.rdf'])
+        .pipe(gulp.dest(GH_PAGES_DIR + EXTENSION_SRC_DIR + 'firefox/')),
+    gulp.src([CHROME_BUILD_DIR + '*.crx',
+              FIREFOX_BUILD_DIR + '*.rdf'])
```

Comment:
I know that you're just moving the old code over, but after looking at this line:
Why are we copying files, relevant to the `FIREFOX` addon, into a folder that ought to contain the `CHROMIUM` extension!?